regression/cuda/benchmarks/015_miscfail4/test.desc

```diff
@@ -2,4 +2,4 @@ CORE
 main.cu
 --unwind 9 --force-malloc-success --state-hashing --context-bound 2 --data-races-check --no-bounds-check
 
-^VERIFICATION FAILED$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Do you have a failing copy for this one?

Reply:
No need, it's the same type as 014